test/overlay/notes.lua

```diff
@@ -0,0 +1,245 @@
+local gui = require('gui')
+local overlay = require('plugins.overlay')
+local guidm = require('gui.dwarfmode')
+local utils = require('utils')
+local notes_textures = reqscript('notes').textures
+
+local waypoints = df.global.plotinfo.waypoints
+local map_points = df.global.plotinfo.waypoints.points
+
+config = {
+    target = 'notes',
+    mode = 'fortress'
+}
```

Comment:
you can automatically run test setup and cleanup by defining `config.wrapper`. Example: test/assign-minecarts.lua or test/plugins/orders.lua

Reply:
I feel such helpers only make the code harder to understand, I prefer explicit definition for setup and cleanup per test.